storops/unity/resource/lun.py

```diff
@@ -388,11 +388,17 @@ def _do_check_move_session(move_session_id):
                 return False
 
         clz = storops.unity.resource.move_session.UnityMoveSession
-        is_compressed = self.is_data_reduction_enabled
-
+        is_compressed = True if provision == 'compressed' else \
```

Comment:
Move these codes out of storops, get the values of is_thin and is_compressed in cinder driver client.py, and pass them when invoking migrate method.

Reply:
Move this part to cinder code as suggested.